ReactiveFeedback/Feedback.swift

```diff
@@ -3,12 +3,22 @@ import ReactiveSwift
 import enum Result.NoError
 
 public struct Feedback<State, Event> {
-    public let events: (Scheduler, Signal<State, NoError>) -> Signal<Event, NoError>
+    let events: (Scheduler, Signal<State, NoError>) -> Signal<Event, NoError>
     
+    /// Creates an arbitrary Feedback, by transforming a sequence of State to a sequence of Events that mutate the State
+    ///
+    /// - parameters:
+    ///    - events: A closure which transforms the Signal<State, NoError> to the Signal<Event, NoError>
     public init(events: @escaping (Scheduler, Signal<State, NoError>) -> Signal<Event, NoError>) {
         self.events = events
     }
 
+    /// Creates a Feedback which will perform effects when `query` exists (not nil)
+    /// and is different from previous, otherwise, cancels previously performed effects.
```

Comment:
`Creates a Feedback which will perform effects when `query` exists (i.e. is not nil)
and is different from the previous one, otherwise, cancels the previously performed effects.`